### PR TITLE
Make it work in the Amsterdam Azure environment

### PR DESCRIPTION
**Pull request by IanR01**

### CODE REVIEW

File: middleware/auth.js [hunk 1]
```diff
@@ -59,9 +57,12 @@ const ensureRights = (req, res, next) => {
     next();
   } else {
     req.session.destroy(() => {
+      let url = '/admin/login'
```
Comment (TerryvanWalen) [inline middleware/auth.js]: Is there a reason you don't create the complete url in one go? I see you do it in the rest of the code:

```
let url = `${process.env.APP_URL}/admin/users
```

Even better would be to use a function to create the absolute url (that correctly handles url concatenation). To mitigate problems with leading/trailing/absent '/' in the APP_URL or path.

File: public/js/main.js [hunk 2]
```diff
@@ -129,7 +129,7 @@ $(function () {
        { data: "email" },
        { data: "id",
          render: function (val) {
-           return '<a href="/admin/user/'+val+'" target="_blank">Edit</a>'
+           return '<a href="./user/' + val + '" target="_blank">Edit</a>'
```
Comment (TerryvanWalen) [inline public/js/main.js:132]: Shouldn't this be an absolute url too?

**Review by TerryvanWalen — DISMISSED** [review]
Looking good. Would probably choose either using the `env.app_url`, the `appUrl` shorthand I see (imported from a settings/main file) or, my preferred method, a `getAbsoluteUrl()` or similar function to create (and correctly concat) the app_url and relative path.

File: public/js/main.js (hunk 2 above)
Comment (IanR01) [reply public/js/main.js:132]: This js code is loaded client side in the browser, where the APP_URL env var is not known (in the current situation). So I think this is the easiest solution.

File: middleware/auth.js (hunk 1 above)
Comment (IanR01) [reply middleware/auth.js]: Yes you are right, forgot these two.

About the function to create a clean absolute URL you are also right, but will do this later.

**Review by thmsv1990 — APPROVED** [review]

**Pull request merged**